languages/common-lisp/exercises/concept/lists/.docs/after.md

```diff
@@ -0,0 +1,34 @@
+## Summary
+
+Lists are a very common datatype in Common Lisp. They can be constructed via `list` or `cons` or even just quoting `'(a b c)`. Elements of a list can be accessed by index via `nth`, the helper functions `first` through `tenth`.
```

Comment:
As we want the `after.md` document to contain the information that is in the `introduction.md` document, maybe expand on this a bit to show some examples of these functions?